From 474cc44934eb915f97dd8ba876e2256c40b4ea9a Mon Sep 17 00:00:00 2001
From: worldofpeace <worldofpeace@protonmail.ch>
Date: Fri, 13 Dec 2019 10:45:27 -0500
Subject: [PATCH 1/3] Default DBUS_SYS_DIR to ${datadir}/dbus-1/system.d

Since D-Bus 1.9.18 configuration files installed by third-party should
go in share/dbus-1/system.d. The old location is for sysadmin overrides.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 2b847d5..4deeb78 100644
--- a/configure.ac
+++ b/configure.ac
@@ -128,7 +128,7 @@ AC_ARG_WITH(dbus-services,
 if ! test -z "$with_dbus_sys" ; then
         DBUS_SYS_DIR="$with_dbus_sys"
 else
-        DBUS_SYS_DIR='${sysconfdir}/dbus-1/system.d'
+        DBUS_SYS_DIR='${datadir}/dbus-1/system.d'
 fi
 AC_SUBST(DBUS_SYS_DIR)
 
From 77d41dd146f9ad64656fab9f3fc9a983b8109f25 Mon Sep 17 00:00:00 2001
From: Christian Hesse <mail@eworm.de>
Date: Thu, 9 Jan 2020 00:27:23 +0100
Subject: [PATCH 2/3] Merge pull request #303 from eworm-de/dbus-dir

actually use the configured dbus directory
---
 plugins/datetime/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins/datetime/Makefile.am b/plugins/datetime/Makefile.am
index d705b60..fed59df 100644
--- a/plugins/datetime/Makefile.am
+++ b/plugins/datetime/Makefile.am
@@ -1,5 +1,5 @@
 dbus_servicesdir = $(datadir)/dbus-1/system-services
-dbus_confdir = $(sysconfdir)/dbus-1/system.d
+dbus_confdir = $(DBUS_SYS_DIR)
 polkitdir = $(datadir)/polkit-1/actions
 
 dbus_services_in_files = org.mate.SettingsDaemon.DateTimeMechanism.service.in
From c1519c5669112499b889fe4c1d5373aaa633b0d6 Mon Sep 17 00:00:00 2001
From: monsta <monsta@inbox.ru>
Date: Fri, 10 Jan 2020 14:09:28 +0300
Subject: [PATCH 3/3] require DBUS >= 1.10.0

needed after 474cc44934eb915f97dd8ba876e2256c40b4ea9a
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 4deeb78..094ac04 100644
--- a/configure.ac
+++ b/configure.ac
@@ -318,7 +318,7 @@ AC_SUBST(NSS_DATABASE)
 # ---------------------------------------------------------------------------
 
 POLKIT_REQUIRED=0.97
-DBUS_REQUIRED=1.1.2
+DBUS_REQUIRED=1.10.0
 
 # PolicyKit detection; defaults to 'auto' (use it if it's available)
 #
